.github/workflows/create-urgent-hotfix-img.yml

```diff
@@ -0,0 +1,301 @@
+# type: CI
+# owner: @camunda/monorepo-devops-team
+
+# Description:
+# This GitHub Actions workflow creates Docker images for urgent hotfixes.
+# See the documentation about it at https://confluence.camunda.com/x/dwMNE
+#
+# It is triggered manually via workflow dispatch
+# and allows building specific components with versions from either a branch or a specific commit SHA.
+#
+# Usage:
+# - Input format for components: "component:version,component:version" (e.g., "camunda:8.6.1,operate:8.6.1")
+# - Either branch OR commit must be specified (commit takes precedence)
+# - Commit must be a FULL 40-character SHA (short SHAs are not supported)
+# - Support ticket ID is required for proper tagging (e.g. SUPPORT-12345, JIRA-123, ticket-456)
+# - Registry is fixed to: registry.camunda.cloud/hotfixes (not configurable)
+# - Platforms are not configurable, they are by default set to linux/amd64 and linux/arm64
+#
+# The workflow validates inputs, builds the requested images, and pushes them to the Camunda hotfix registry following
+# the naming convention: registry.camunda.cloud/hotfixes/<image>:<application-version>-<support-issue-id>
```

Comment:
to be updated after https://github.com/camunda/camunda/pull/39491/files#r2452690290